.meta/dietpi-imager

```diff
@@ -246,12 +273,135 @@ Do you want to overwrite or backup the existing file to /root/$OUTPUT_IMG_NAME.$
 			# Rename and move to /root, if source image != output image already
 			[[ /root/$OUTPUT_IMG_NAME.$OUTPUT_IMG_EXT != $FP_SOURCE_IMG ]] && mv "$FP_SOURCE_IMG" $OUTPUT_IMG_NAME.$OUTPUT_IMG_EXT
 
-		# Drive source
-		else
+		# Drive source, clone with dd
+		elif [[ $CLONING_TOOL == 'dd' ]]; then
 
 			G_DIETPI-NOTIFY 2 "Creating final image with actually used size: $(( $IMAGE_SIZE / 1024 / 1024 + 1 )) MiB"
 			dd if=$FP_SOURCE of=$OUTPUT_IMG_NAME.$OUTPUT_IMG_EXT bs=1M status=progress count=$(( $IMAGE_SIZE / 1024 / 1024 + 1 ))
 
+		# Drive source, clone with Clonezilla
+		else
+
+			G_DIETPI-NOTIFY 2 'Creating final image with Clonezilla'
+			# Install required packages
+			G_AG_CHECK_INSTALL_PREREQ clonezilla partclone xorriso isolinux syslinux-common unzip curl || Exit_On_Fail
+			# TODO: 'partimage' is also required by Clonezila, but the package for buster is missing.
```

Comment:
From Debian, "partimage" is only recommended but not required. From some changelog I found (quick and dirty), it looks like it has been deprecated for Clonezilla for exactly the missing Buster package: https://www.afterdawn.com/software/version_history.cfm/clonezilla

Reply:
Yep, partimage seems to be abandoned (it has not been updated since 2010) and partclone supports all the needed file systems. Nevertheless, if for whatever reason partclone fails, Clonezilla will try to use partimage instead. That's because of the -q2 parameter (use partclone to save partition(s) i.e. partclone > partimage > dd). So there's a tiny chance that if partimage is not installed ocs-sr could end with an error, but that's already been taken into account.
We can remove this note. Probably Clonezilla will drop support for partimage soon.

Reply:
@sal666 
As we install `partclone` explicitly and error-handled above, I guess we do not need to think anymore about this. If `partclone` fails for a different reason than being not available, then `partimage` most likely will fail as well, and the root must be fixed before retrying just with another tool and probably making things worse by this 😉.

Reply:
Removed.